build/conda/installer/post_install.bat

```diff
@@ -1,28 +1,31 @@
 @echo off
-echo Running post-installation environment setup.
-REM Full path to the Python executable inside the constructed environment
-SET PYTHON_EXEC="%PREFIX%\python.exe"
+echo Installing environment, this may take a few moments... Watch for changes in %PREFIX%\post_install_log.txt
+
+cd %PREFIX%
+
+PATH %PREFIX%;%PREFIX%\Scripts;%PREFIX%\Library\bin;%PATH%
 SET LOG_FILE="%PREFIX%\post_install_log.txt"
 
-REM Use the specific Python that comes bundled with the installer
-"%PYTHON_EXEC%" -m pip install -U openbb[all] openbb-cli openbb-platform-pro-backend >> "%LOG_FILE%" 2>&1
+python -m pip install -U pip >> "%LOG_FILE%" 2>&1
+pip -m pip install -U setuptools >> "%LOG_FILE%" 2>&1
+pip install -U -r requirements.txt >> "%LOG_FILE%" 2>&1
 IF ERRORLEVEL 1 (
     echo %date% %time% "Error during post-installation: pip install failed." >> %LOG_FILE%
     exit /b 1
 ) ELSE (
     echo %date% %time% "pip install completed successfully." >> %LOG_FILE%
 )
 
-REM Build OpenBB's python interface
-"%PYTHON_EXEC%" -c "import openbb; openbb.build()" >> "%LOG_FILE%" 2>&1
+echo Package installation completed successfully. Building OpenBB's Python interface...
+
+python -c "import openbb; openbb.build()" >> "%LOG_FILE%" 2>&1
 IF ERRORLEVEL 1 (
-    echo %date% %time% "Error during post-installation: building OpenBB's python interface failed."  >> %LOG_FILE%
+    call :log_with_timestamp "Error during post-installation: building OpenBB's Python interface failed."
```

Comment:
In the initial bat script the function `log_with_timestamp` was broken and was breaking the script. can you double-check that these lines work? if we removed the log_with_timestamp you can revert to vanilla `echo %date% %time% "Error during post-installation: building OpenBB's python interface failed." >> %LOG_FILE%`

Reply:
Yes, I fixed the script. The issue was that log_with_timestamp terminated the script whenever it was called, so it would never advance beyond the first call.